specification/apicenter/resource-manager/Microsoft.ApiCenter/preview/2024-03-15-preview/apicenter.json

```diff
@@ -3954,6 +3954,36 @@
         "lifecycleStage"
       ]
     },
+    "ApimSource": {
```

Comment:
>ApimSource [](http://example.com/codeflow?start=5&length=10)

this is not used in this swagger of this API version. is this still needed ?

Reply:
Good catch! Removed.